market/connector/docusign/README.md

```diff
@@ -1 +1,17 @@
-Integrate eSignatures into your application in minutes. DocuSign's secure and award-winning eSignature makes requesting signatures, automating forms, and tracking documents directly from your app easy.
\ No newline at end of file
+Integrate eSignatures into your application in minutes. DocuSign's secure and award-winning eSignature makes requesting signatures, automating forms, and tracking documents directly from your app easy.
+
+## Setup
+
+1. Create a free DocuSign developer account: https://account-d.docusign.com/#/username
+1. Create a new `application`: https://admindemo.docusign.com/authenticate?goTo=apiIntegratorKey
```

Comment:
Seems to work, but I didn't know that this works if every numbered step is 1 :)

Reply:
It's a markdown feature: and a quite handy one if you don't want to spend to much time when adding new points.
some editors, such as my favourite offline editor Hardoopad  https://github.com/rhiokim/haroopad , do also preview and render such numbered lists.

Reply:
VS Code can preview too. But this one looks cool too 👍